docs/geometry/polar-coordinate.md

```diff
@@ -1,24 +1,24 @@
 author: Ir1d, HeRaNO, Chrogeek, abc1763613206
 
-## 极坐标与极坐标系
+## Polar coordinates and polar coordinate system
 
-（为人教版高中数学选修 4-4 内容）
+> This paragraph is taken from chapter 4-4 high school mathematics electives, people's education edition.
```

Comment:
This is the content of the book *Senior High School Math: Elective 4-4* by *People's Education Press*